test/testApolloServer.js

```diff
@@ -185,6 +199,22 @@ describe('ApolloServer', () => {
       );
     });
   });
+
+  it('will append resolvers to a js schema', () => {
+    const app = express();
+    const jsServer = apolloServer({
+      schema: testJSSchema,
+      resolvers: testResolvers
+    });
+    app.use('/graphql', jsServer);
+    return request(app).get(
+      '/graphql?query={stuff}'
+    ).then((res) => {
+      const fields = testJSSchema.getType("RootQuery").getFields();
+      expect(fields['usecontext']['resolve']).to.exist;
```

Comment:
Is there a reason you're testing that the resolver was attached to the schema, instead of testing that it actually works as expected end-to-end? I think it might be more future-proof to just make sure that the server's response is what you expected (which could only happen if the resolve function is attached).